config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root 'employees#index'
+  get '/administrators', to: 'administrators#index'
```

Comment:
you could remove this route because you're using administrator resources

Reply:
Ok, I will do the corrections, thanks :)